TFExplorer/settings.py

```diff
@@ -165,3 +166,222 @@
             " You have kept DEBUG=FALSE without a local_settings. \
             It is recommended to keep prod settings seperate in local_settings.py "
         )
+
+
+#drf settings
+from django.conf import settings
+from django.test.signals import setting_changed
+from django.utils.module_loading import import_string
+from rest_framework import ISO_8601
```

Comment:
All the imports should be at the top of the file

Reply:
Also, many of these imports may not be required after removing the lines in the below comment. Remove those imports